packages/paste-core/primitives/text/src/types.ts

```diff
@@ -14,14 +7,19 @@ import type {
   SpaceProps,
   TypographyProps,
   VerticalAlign,
+  CursorProperty,
+  OutlineProperty,
+  TextTransformProperty,
+  TransitionProperty,
+  FontVariantNumericProperty,
```

Comment:
Praise: Both this and Box consuming types from style-props rather than directly depending on csstype makes a tonne of sense